bench/GEMMsBenchmark.cc

```diff
@@ -224,12 +224,12 @@ void performance_test(const int M, const int N, const int K) {
     // printMatrix(matrix_op_t::NoTranspose,
     // Cint8_fb.data(), m, n, n, "C fb");
 
-#ifdef FBGEMM_MEASURE_TIME_BREAKDOWN
-    cout << ", " << setw(16) << total_packing_time / (double)NITER / 1e3 << ", "
+    if (timebreak)
+      cout << ", " << setw(16) << total_packing_time / (double)NITER / 1e3 << ", "
          << setw(16) << total_kernel_time / (double)NITER / 1e3 << ", "
          << setw(16) << total_postprocessing_time / (double)NITER / 1e3 << ", "
          << setw(16) << total_run_time / (double)NITER / 1e3;
```

Comment:
You may want to print out `total_computing_time` here, to silence the warning-error:

```
/Users/runner/work/FBGEMM/FBGEMM/bench/GEMMsBenchmark.cc:160:34: error: variable 'total_computing_time' set but not used [-Werror,-Wunused-but-set-variable]
    double computing_time = 0.0, total_computing_time = 0.0;
```

It appears that `+=` is just setting, but not using the variable (see [here](https://stackoverflow.com/questions/23998283/variable-warning-set-but-not-used))

Reply:
done